rpc/src/module/debug.rs

```diff
@@ -5,19 +5,19 @@ use std::time;
 #[rpc(server)]
 pub trait DebugRpc {
     #[rpc(name = "jemalloc_profiling_dump")]
-    fn jemalloc_profiling_dump(&self) -> Result<()>;
+    fn jemalloc_profiling_dump(&self) -> Result<String>;
 }
 
 pub(crate) struct DebugRpcImpl {}
 
 impl DebugRpc for DebugRpcImpl {
-    fn jemalloc_profiling_dump(&self) -> Result<()> {
+    fn jemalloc_profiling_dump(&self) -> Result<String> {
         let timestamp = time::SystemTime::now()
             .duration_since(time::SystemTime::UNIX_EPOCH)
             .unwrap()
             .as_secs();
         let filename = format!("ckb-jeprof.{}.heap\0", timestamp);
-        ckb_memory_tracker::jemalloc_profiling_dump(filename);
-        Ok(())
+        ckb_memory_tracker::jemalloc_profiling_dump(filename.clone());
```

Comment:
**(I haven't tested.)** I think this `clone()` is unnecessary, pass a `&mut str` should be enough.

https://github.com/nervosnetwork/ckb/blob/3f27eae7998ee24b282da54eb8d9cbe5b5e2d4cc/util/memory-tracker/src/jemalloc.rs#L7-L13

https://github.com/gnzlbg/jemallocator/blob/c27a859e98e3cb790dc269773d9da71a1e918458/jemalloc-sys/src/lib.rs#L470-L476
(I don't know why this link can't preview, because it has been transferred?)

Reply:
Since u have to fix the lint errors, remove this `clone()`?